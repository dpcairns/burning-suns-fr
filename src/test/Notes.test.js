import React from 'react';
import { shallow } from 'enzyme';
import Test from '../Notes';


describe('First React component test with Enzyme', () => {
    it('renders without crashing', () => {
      const wrapper = shallow(<Test />);
      expect(wrapper).toMatchSnapshot()
     });
 });