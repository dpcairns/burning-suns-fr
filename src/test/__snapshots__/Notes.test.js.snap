// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First React component test with Enzyme renders without crashing 1`] = `
<div>
  <WithStyles(ForwardRef(ExpansionPanel))
    expanded={false}
    onChange={[Function]}
  >
    <WithStyles(ForwardRef(ExpansionPanelSummary))
      aria-controls="panel1bh-content"
      expandIcon={<UNDEFINED />}
      id="panel1bh-header"
    >
      <WithStyles(ForwardRef(Typography))
        className="makeStyles-heading-8"
      >
        Add a Note or a Wish
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        className="makeStyles-secondaryHeading-9"
      >
        Expand to enter details
      </WithStyles(ForwardRef(Typography))>
    </WithStyles(ForwardRef(ExpansionPanelSummary))>
    <WithStyles(ForwardRef(ExpansionPanelDetails))
      mx="auto"
    >
      <Styled(MuiBox)
        bgcolor="background.paper"
        mx="auto"
        p={1}
      >
        <NoteForm
          updateNotes={[Function]}
        />
      </Styled(MuiBox)>
    </WithStyles(ForwardRef(ExpansionPanelDetails))>
  </WithStyles(ForwardRef(ExpansionPanel))>
  <WithStyles(ForwardRef(Divider)) />
  <p>
    Add some notes!
  </p>
</div>
`;
